design/ironic-standalone-operator.md

```diff
@@ -0,0 +1,369 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# Ironic Standalone Operator
+
+## Status
+
+implementable
+
+## Summary
+
+This proposal discussed the [ironic-standalone-operator][ir-op] project that
+was written using inspiration from OpenShift's
+[cluster-baremetal-operator][cbo]. The project is already under the Metal3's
+umbrella, so this document serves to describe its design goals, future plans
+and the rough API shape.
+
+[ir-op]: https://github.com/metal3-io/ironic-standalone-operator
+[cbo]: https://github.com/openshift/cluster-baremetal-operator
+
+## Motivation
+
+Ironic is not trivial to install and operate. Also we provide
```

Comment:
"Even if we provide" ?

Reply:
Also-vs-Although 🤦🏽 Will fix if I need to update.